fakesmtp/fakesmtp.go

```diff
@@ -0,0 +1,98 @@
+package fakesmtp
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+	"time"
+)
+
+var LastMessage string
```

Comment:
I would probably get rid of this and have a `Messages()` method that returns the messages from the client, as well as a `Clear()` function that clears out all the saved messages between tests.